src/App.jsx

```diff
@@ -1,11 +1,17 @@
-import './App.css';
-import {Errbound} from './components/errBoundary';
-
+import "./App.css";
+import Errboundary from "./components/ErrBoundary/ErrBoundary";
+import { Counter } from "./components/ErrBoundary/Counter";
+import { Decreement } from "./components/ErrBoundary/Decreement";
```

Comment:
```suggestion
import { Decrement } from "./components/ErrBoundary/Decrement";
```
Not important but nice to have [Nit]